core/src/main/scala/io/snappydata/Literals.scala

```diff
@@ -18,168 +18,8 @@ package io.snappydata
 
 import scala.reflect.ClassTag
 
-import com.gemstone.gemfire.internal.shared.SystemProperties
-import io.snappydata.collection.ObjectObjectHashMap
-
-import org.apache.spark.sql.collection.Utils
```

Comment:
Ok. So you just transferred all these Constants from Literals to Constants and the foldable functions to the BaseParser. Just proper places for defining them or some other reason to move?